app/views/content_items/answer.html.erb

```diff
@@ -1,15 +1 @@
-<% content_for :simple_header, true %>
-<% content_for :title, @content_item.title %>
-
-<div class="grid-row sidebar-with-body">
-  <div class="column-two-thirds">
-    <%= render 'govuk_component/title',
-        title: @content_item.title %>
-    <%= render 'govuk_component/govspeak',
-        content: @content_item.body,
-        direction: page_text_direction,
-        rich_govspeak: true
-    %>
-  </div>
-  <%= render 'shared/related_items', content_item: @content_item  %>
-</div>
+<%= render 'content_items/short_text/just_govspeak' %>
```

Comment:
Could we avoid the `short_text` nesting? Technically everything in government-frontend is short text except HTML pubs.